packages/@okta/i18n/src/properties/login.properties

```diff
@@ -718,6 +718,11 @@ cert.authentication.title = Certificate authentication
 ##                       OIE strings            ##
 ##################################################
 
+## Generic Authenticator Messages
+# {0} refers to the authenticator name (Duo, Symantec VIP, etc.)
+oie.authenticator.enroll.title = Set up {0}
+oie.authenticator.challenge.title = Verify with {0}
```

Comment:
Yes, if the key exists for the exact same string then reuse it.

Reply:
Is it better to have unique keys per authenticator for customization support ?

Reply:
> Is it better to have unique keys per authenticator for customization support ?

I'm really torn on this. On one hand - I love the idea of grouping all titles under a single key (assuming the values are equivalent), but we do lose out on customizability. If we do allow these `title` attributes to be customized, do we also want to extend it to the `subTitle`, input fields, and the **Enroll** and **Verify** buttons?

I think to get the answer, we'll need to loop in PM to see what our customizability story is. For now, I'd prefer we try to avoid duplication/redundancies when we can.

Reply:
We have had several discussions with the PM and the ask from is to definitely to have support for customization. I believe this is why we have created individual keys for authenticators.

Reply:
Thanks @nikhilvenkatraman-okta! Will continue the v1 convention of keeping these view-specific.